### PR TITLE
(config) Add forceOverwrite option in ort configuration reference.yml file

### PR DESCRIPTION
**Pull request by dineshr93**
Added option to overwrite existing result files option (forceOverwrite) in ort configuration reference.yml

### CODE REVIEW

File: model/src/main/resources/reference.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@
 # exclusive, so this file is only used to show all options and to validate the configuration.
 
 ort:
+  # To overwrite if the reports (analyzer-result.yml) already exists
+  forceOverwrite: true
```
Comment (mnonnenmacher) [inline model/src/main/resources/reference.yml:23]: @dineshr93 Thank you for the contribution. Please put these lines in alphabetical order (so before `licenseFilePatterns`) and surround it with single empty lines for a better overview. And please also add this line at the same location to the `OrtConfigurationTest.kt` which verifies that the reference.yml is correctly parsed:
`ortConfig.forceOverwrite shouldBe true`

To pass the functional tests you will have to rebase your branch.
To pass the commitlint check you need to change the title to e.g.:
"feat(config): Add forceOverwrite option to reference.yml"

Also, please start the sentence in the commit message body capitalized and end it with a dot, but actually you could just remove it as to me the title is self contained.

File: model/src/main/resources/reference.yml [hunk 2]
```diff
@@ -19,6 +19,8 @@
 # exclusive, so this file is only used to show all options and to validate the configuration.
 
 ort:
+  # To overwrite if the reports (analyzer-result.yml) already exists
```
Comment (mnonnenmacher) [inline model/src/main/resources/reference.yml:22]: Just use the same wording as for the configuration property in `OrtConfiguration.kt`:
"Force overwriting of any existing output files."

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

**Comment by sschuberth** [comment]
@dineshr93 there's now a merge commit in your PR, which is something that we do not allow. Please rebase instead of merge in order to catch up your PR with upstream changes.

**Comment by sschuberth** [comment]
@dineshr93 are you planning to continue the work on this PR?

**Comment by dineshr93** [comment]
Hi @sschuberth  Please take this pull request https://github.com/oss-review-toolkit/ort/pull/8146  instead of this pull request Thanks

**Comment by sschuberth** [comment]
> Please take this pull request #8146 instead of this pull request

Thanks. However, organizational-wise it would have been easier for us to continue in this PR with rewritten Git history. Please keep that in mind for future contributions.

**Comment by dineshr93** [comment]
> > Please take this pull request #8146 instead of this pull request
> 
> Thanks. However, organizational-wise it would have been easier for us to continue in this PR with rewritten Git history. Please keep that in mind for future contributions.

I am not sure how to remove the merge commit Thats why.. However What would be the steps to remove the merge commit and do debasing while already raised a pull request?

**Comment by sschuberth** [comment]
> What would be the steps to remove the merge commit and do debasing while already raised a pull request?

Just reset our local branch to the commit before the merge commit(s), then rebase the branch onto latest `origin/main` (fetch first), and force-push your branch.

**Pull request closed without merging**